Bindings/bindings.json

```diff
@@ -302,7 +310,8 @@
                     "resource": "EventHub",
                     "required": true,
                     "label": "Service Bus connection",
-                    "help": "This is the connection string for your Service Bus account. Make sure that it has Listen rights."
+                    "help": "This is the connection string for your Service Bus account. Make sure that it has Listen rights.",
```

Comment:
In that case, it should be all lowercase. "service bus" would be the correct form